client/config.go

```diff
@@ -2,8 +2,9 @@ package client
 
 // Config defines Provider Configuration
 type Config struct {
-	ProjectFilter string   `hcl:"project_filter,optional"`
-	ProjectIDs    []string `hcl:"project_ids,optional"`
+	ProjectFilter         string   `hcl:"project_filter,optional"`
+	ProjectIDs            []string `hcl:"project_ids,optional"`
+	ServiceAccountKeyJSON string   `hcl:"service_account_key_json,optional"`
```

Comment:
we can make this a block and then marshal that block into a json and pass it to the client, but I am not sure if that's prettier than the actual json itself. wdyt?

Reply:
How would that look like in the `config.hcl` file? I guess you would have to translate the secrets file (which is json) to HCL?